client/src/redux/sagas/datafiles.sagas.test.js

```diff
@@ -487,8 +488,8 @@ describe('compressFiles', () => {
     });
   };
 
-  it('runs compressFiles saga with success', () => {
-    return expectSaga(compressFiles, createAction('private'))
+  it.skip('runs compressFiles saga with success', () => {
```

Comment:
Extract PR also has these tests skipped

Reply:
Yep. I can remove the changes from the Extract PR and only have them here.